orchestration/dagster_orchestration/hca_orchestration/solids/load_hca/load_table.py

```diff
@@ -1,56 +1,122 @@
 import logging
```

Comment:
This is a fairly sizable refactor of the table loading code. The main points of interest:

1. Reduced the granularity of the solids to avoid the awkward `yield result -> check result to conditionally dispatch downstream solid` pattern. In retrospect that pattern has two related downsides: a) it's difficult to test b) it's difficult to debug operationally. Rolling all of that conditional logic into a single (beefier) python method makes this code less surprising.

2. Switches to consuming a `data_repo_service` which lets us get rid of all the job polling code that was sprinkled throughout.